test/integrations/destinations/google_adwords_remarketing_lists/processor/data.ts

```diff
@@ -79,7 +79,10 @@ export const data = [
               params: {
```

Comment:
We need to add new test cases for the scenarios where consent data is specified.